### PR TITLE
Add ISSUE_TEMPLATE/ru.yaml

### PR DESCRIPTION
**Pull request by ekalinin**
Added translate files for issues in Russian:
- ISSUE_TEMPLATE/ru.yaml

As agreed within the issue:
- https://github.com/kubernetes/minikube/pull/13086#issuecomment-987184278

### CODE REVIEW

**Review by medyagh — COMMENTED** [review]
thank you @ekalinin do u mind using this newer template

https://github.com/kubernetes/minikube/issues/new?assignees=&labels=&template=experimental.yaml

we are moving to this new format for english and I expect other languages to use same

**Comment by ekalinin** [comment]
> thank you @ekalinin do u mind using this newer template
> 
> https://github.com/kubernetes/minikube/issues/new?assignees=&labels=&template=experimental.yaml
> 
> we are moving to this new format for english and I expect other languages to use same

hey @medyagh 

done!

File: .github/ISSUE_TEMPLATE/russian.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+name: Сообщить о проблеме
```
Comment (medyagh) [inline .github/ISSUE_TEMPLATE/russian.yaml:1]: we can remove the ru.md and also lets name ths one ru.yaml.

sposiba ! :)

**Review by medyagh — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/russian.yaml (hunk 1 above)
Comment (ekalinin) [reply .github/ISSUE_TEMPLATE/russian.yaml]: Done!

You're welcome! :)

**Review by medyagh — APPROVED** [review]
sposiba !

**Comment by ekalinin** [comment]
> sposiba !

~~thank you too~~ I vam spasibo! 8)

**Pull request merged**